search.go

```diff
@@ -171,6 +173,24 @@ func (c *Client) DeleteSearchJob(ctx context.Context, searchID string) error {
 	return nil
 }
 
+func (c *Client) UpdateSearchConcurrencySettingsScheduler(ctx context.Context, req *UpdateSearchConcurrencySettingsScheduleReq) error {
```

Comment:
Could you add a comment with a description of what this really does, or even just the text from the splunk docs would be useful.

Reply:
Sure I ll do